app/src/lib/git/cherry-pick.ts

```diff
@@ -8,7 +17,22 @@ export enum CherryPickResult {
    * signal success to the user.
    */
   CompletedWithoutError = 'CompletedWithoutError',
-
+  /**
+   * The cherry pick encountered conflicts while attempting to cherry pick and
+   * need to be resolved by the user can continue.
```

Comment:
Nit: I think this is what you meant? 😳 
```suggestion
   * need to be resolved before the user can continue.
```